src/csharp/Microsoft.Spark/Hadoop/Fs/FileSystem.cs

```diff
@@ -55,6 +55,14 @@ public bool Delete(string path, bool recursive = true)
             return (bool)_jvmObject.Invoke("delete", pathObject, recursive);
         }
 
+        public bool Exists(string path)
```

Comment:
Could you please add a documentation comment describing the method?  You can copy its content from open-source `org.apache.hadoop.fs.FileSystem`.

Reply:
Done